docs/scripts/formattedTSDemos.js

```diff
@@ -9,7 +9,11 @@
  * List of demos or folders to ignore when transpiling
  * Example: "app-bar/BottomAppBar.tsx"
  */
-const ignoreList = ['/pages.ts', 'docs/data/joy/getting-started/templates'];
+const ignoreList = [
+  '/pages.ts',
+  'docs/data/joy/getting-started/templates',
+  'docs/data/base/components/select/UnstyledSelectIntroduction.tsx',
```

Comment:
This demo will create PropTypes and `slotProps.popper` gets wild... here's the snapshot... hence this change for now.. @michaldudak 
<img width="474" alt="Screenshot 2023-05-01 at 14 47 48" src="https://user-images.githubusercontent.com/32841130/235460762-6d88ccaf-e258-4bbd-9c99-7d5c62492d76.png">

Reply:
Let's create an issue for this prop, would be great to know what's causing it.